src/CalendarDate.ts

```diff
@@ -267,6 +267,10 @@ export class CalendarDate {
     return new Date(this.unixTimestampInSeconds * 1000);
   }
 
+  toLocalDate(): Date {
+    return new Date(this.year, this.month - 1, this.day);
+  }
```

Comment:
```suggestion
  toDateLocal(): Date {
    return new Date(this.year, this.month - 1, this.day);
  }
```
i would change the naming to this, to keep in line with the naming of `fromDateLocal`. I would also like to change the `toDate` method to `toDateUTC` to be in line with `fromDateUTC`.

Reply:
I have renamed the methods. This now has a breaking change.

Reply:
@bjaminp i also thought about this renaming of toDate, since its a breaking change. But i would still go with a minor version incease instead of a major. 
Reasons for me are:
- We only removed a method and did not change existing logic. So if you use typescript in your project or have any kind of tests if you are using JS then this will be catched very easily in pipelines.
- at least in our organization toDate is no really used, so there won't be a lot of changes.
- The packages currently is not used by a lot of people outside of our organization.

Reply:
As a user of the package, I would have preferred a major version, as we rely on the SemVer semantics that a minor version is when you add functionality in a backward compatible manner. It just avoids surprises when using tools such as renovate.